coresimd/acle/simd32.rs

```diff
@@ -0,0 +1,60 @@
+//! # References
+//!
+//! - Section 8.5 "32-bit SIMD intrinsics" of ACLE
+//!
+//! Intrinsics that could live here
+//!
+//! - __ssat16
+//! - __usat16
+//! - __sxtab16
+//! - __sxtb16
+//! - __uxtab16
+//! - __uxtb16
+//! - __qsub8
+//! - __sadd8
+//! - __shadd8
+//! - __shsub8
+//! - __ssub8
+//! - __uadd8
+//! - __uhadd8
+//! - __uhsub8
+//! - __uqadd8
+//! - __uqsub8
+//! - __usub8
+//! - __usad8
+//! - __usada8
+//! - __qadd16
+//! - __qasx
+//! - __qsub16
+//! - __sadd16
+//! - __sasx
+//! - __shadd16
+//! - __shasx
+//! - __shsax
+//! - __shsub16
+//! - __ssax
+//! - __ssub16
+//! - __uadd16
+//! - __uasx
+//! - __uhadd16
+//! - __uhasx
+//! - __uhsax
+//! - __uhsub16
+//! - __uqadd16
+//! - __uqasx
+//! - __uqsax
+//! - __uqsub16
+//! - __usax
+//! - __usub16
+//! - __smlad
+//! - __smladx
+//! - __smlald
+//! - __smlaldx
+//! - __smlsd
+//! - __smlsdx
+//! - __smlsld
+//! - __smlsldx
+//! - __smuad
+//! - __smuadx
+//! - __smusd
+//! - __smusdx
```

Comment:
You need to add a separate module which just contains `__usat` and `__ssat`. These two instructions are supported on ARMv6+ for A profile and ARMv7-M (as part of the base v7, not v7EM).

You don't need to fill in the implementation immediately, a placeholder like above would be enough.